test/features/support/GithubAPI/Issues/IssuesTestSteps.js

```diff
@@ -0,0 +1,39 @@
+const defaults = require('superagent-defaults');
+
+const agent = defaults();
+const { Given, When, Then } = require('cucumber');
+const chai = require('chai');
+const { setAuthorizationHeaders } = require('../../helpers/helpers');
+
+const { expect } = chai;
+
+const urlBase = 'https://api.github.com';
+
+let githubRepositoryName;
+
+const query = { title: 'API Created Issue' };
+const updateQuery = { body: 'API Created Issue new body' };
+
+Given('a public repository of the account {string}', (repository) => {
+  setAuthorizationHeaders(agent);
```

Comment:
I'm seeing that you are setting this in all the scenarios, what about to create a step or move into a hook?